boards/DCM/Src/App/states/App_FaultState.c

```diff
@@ -4,6 +4,53 @@
 
 #include "App_SharedMacros.h"
 
+#define CLEAR_INV_FAULT_PARAM_ADDRESS (20U)
+#define WRITE_CMD (1U)
+#define CLEAR_INV_FAULT_MSG()                                          \
+    {                                                                  \
+        .d1_parameter_address_command = CLEAR_INV_FAULT_PARAM_ADDRESS, \
+        .d2_read_write_command = WRITE_CMD, .d3_data_command = 0       \
```

Comment:
nit
```suggestion
        .d2_read_write_command = WRITE_CMD, .d3_data_command = 0U       \
```

Reply:
Tbh I'm not a fan of putting small nits in PRs now unless the style is downright unconventional. I think it's taking away from the essence of FW which is to implement some intended functionality

Reply:
in the case above it looks like a function call, which is unusual. i think its also nice to be explicit that the variable is unsigned

Reply:
The function call I took out of using the ESP-IDF a lot. I can put U on every unsigned literal but then that should be put in the coding guidelines in the README

Reply:
done